pow/randomx/src/lib.rs

```diff
@@ -74,6 +81,7 @@ unsafe impl WithCacheMode for WithFullCacheMode {
 pub enum WithLightCacheMode { }
 unsafe impl WithCacheMode for WithLightCacheMode {
 	fn has_dataset() -> bool { false }
+    fn has_large_pages(_config: &Config) -> bool { false }
```

Comment:
Indentation

Reply:
Yes, not sure why but vim is configured to use space tab on rust language files - well i lost some times but i've eventually fixed it